src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -424,6 +425,32 @@ private Future<JsonObject> buildBaseItemRecordJsonObject(CompositePoLine compPOL
       });
   }
 
+  private Future<JsonObject> buildBaseItemRecordJsonObject(CompositePoLine compPOL, BindItem bindItem, RequestContext requestContext) {
+    var holdingLocation = compPOL.getLocations().stream().filter(location -> location.getHoldingId() != null).findAny();
+    if (holdingLocation.isEmpty()) {
+      throw new IllegalArgumentException("Holding Id must not be null");
+    }
+    return InventoryUtils.getLoanTypeId(configurationEntriesCache, inventoryCache, requestContext)
```

Comment:
we don't use loanTypeId from cache, instead we set it from bind item. So this call should be deleted